Source/WebCore/Modules/model-element/HTMLModelElement.idl

```diff
@@ -40,6 +40,9 @@
 
     [Reflect] attribute boolean interactive;
 
+    [Conditional=MODEL_PROCESS, EnabledBySetting=ModelProcessEnabled, CEReactions=NotNeeded] readonly attribute DOMPointReadOnly boundingBoxCenter;
```

Comment:
I'm a little uneasy on having them as 2 separate attributes. Have you considered making it a single `boundingBox` with `{center: domPoint1, extent: domPoint2}` ?